application/pendingtx/pendingindex/pendingtxindexer_test.go

```diff
@@ -0,0 +1,100 @@
+package pendingindex
+
+import (
+	"context"
+	"github.com/alicenet/alicenet/consensus/db"
+	"github.com/alicenet/alicenet/utils"
+	"github.com/dgraph-io/badger/v2"
+	"github.com/stretchr/testify/assert"
+	"testing"
+)
+
+func createDb(t *testing.T) *db.Database {
+	t.Helper()
+	rawEngineDb, err := utils.OpenBadger(context.Background().Done(), "", true)
```

Comment:
There's also [SetupBadgerDatabase](https://github.com/alicenet/alicenet/blob/de8ead25eb422009262236c8def694f59343cb97/internal/testing/environment/environment.go#L11) if that helps.